Services/Skill/classes/class.ilSkillSelectorGUI.php

```diff
@@ -32,7 +32,6 @@ class ilSkillSelectorGUI extends ilVirtualSkillTreeExplorerGUI
     protected $select_gui = "";
     protected string $select_cmd = "";
     protected string $select_par = "";
-    protected bool $select_multi = false;
```

Comment:
@alex40724 I had to move this to the base class, because it is also used there, and without my changes this was dynamically declared in the base class.